move-mutator/src/mutate.rs

```diff
@@ -136,7 +136,14 @@ fn traverse_function(
     let function_name = function.get_name_str();
     let filename = function.module_env.get_source_path();
 
-    // Check if function is included in individual configuration.
+    // Check if a function is included in the general project configuration.
+    if let IncludeFunctions::Selected(funcs) = &conf.project.mutate_functions {
```

Comment:
Just a thought - if there are `include_functions` for individual configuration and `mutate_functions` for general configuration, maybe we should unify that.
You can change the individual configuration to contain the same variable name for functions to be mutated.

Reply:
I think I misunderstood the initial meaning behind the comment, so I optimized this code a bit.
Only then I renamed these variables 😄  

Done